pandas/core/internals/blocks.py

```diff
@@ -2769,7 +2769,7 @@ def make_block(values, placement, klass=None, ndim=None, dtype=None):
 # -----------------------------------------------------------------
 
 
-def _extend_blocks(result, blocks=None):
+def extend_blocks(result, blocks=None):
```

Comment:
btw, we i think you can remove this at this point (assuming all of the return values are List[Block])